libs/models/src/lib/car.entity.ts

```diff
@@ -0,0 +1,81 @@
+import { CompletedAt } from "./base.entity";
+import { ApiModelProperty } from "@nestjs/swagger/dist/decorators/api-model-property.decorator";
+import { Column, Entity, PrimaryGeneratedColumn } from "typeorm";
+import { ICar, Location } from "@hems/interfaces";
+
+
+
+@Entity('cars')
+export class Car extends CompletedAt implements ICar {
+    @ApiModelProperty({ example: '123e4567-e89b-12d3-a456-426614174000' })
+    @PrimaryGeneratedColumn('uuid')
+    carId!: string;
+
+    @ApiModelProperty ({ example: '2345'})
+    @Column ({ length: 254 })
```

Comment:
```suggestion
    @Column ()
```
No need to specify the length